### PR TITLE
Ap/prcss50/add requirements and details of start process

### PR DESCRIPTION
**Pull request by ANGIECATHERINEP**
Requirements are added (consuming the backend service) in the scheduled and on-demand cards
image

The pre-validate requirements modal is added and within this the more details modal
image

The details of the scheduled and on-demand cards are added
image

### CODE REVIEW

**Review by LinconOssa — APPROVED** [review]
Meets requirements

File: src/pages/startProcess/tabs/onDemand/components/OnDemandRequirements/interface.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import { MdInfoOutline } from "react-icons/md";
+import { SkeletonLine } from "@inubekit/skeleton";
+import { Tag } from "@inubekit/tag";
+import { Text } from "@inubekit/text";
+import { Stack } from "@inubekit/stack";
+
+import { appearances } from "@pages/startProcess/types";
+import { RequirementsModal } from "@src/components/modals/requirementsModal";
```
Comment (Dlfp) [inline src/pages/startProcess/tabs/onDemand/components/OnDemandRequirements/interface.tsx:8]: usar el alias

File: src/pages/startProcess/tabs/onDemand/components/OnDemandRequirements/interface.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import { Tooltip } from "@src/design/feedback/Tooltip";
+import { tokens } from "@src/design/tokens";
```
Comment (Dlfp) [inline src/pages/startProcess/tabs/onDemand/components/OnDemandRequirements/interface.tsx:10]: usar el alias

File: src/pages/startProcess/tabs/onDemand/components/OnDemandRequirements/interface.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import {
+  IGeneralStatusRequirementResponse,
+  IProcessRequirementResponse,
+} from "@src/types/statusRequeriments.types";
+import { IData } from "@src/components/modals/requirementsModal/types";
```
Comment (Dlfp) [inline src/pages/startProcess/tabs/onDemand/components/OnDemandRequirements/interface.tsx:15]: en todas parastes donde se usar @src usar el alias correspondiente

**Pull request merged**